Source/UI/NewUI/BillingForm/Default Implementation/Elements/Error/DefaultErrorInputLabelStyle.swift

```diff
@@ -2,11 +2,12 @@ import UIKit
 
 struct DefaultErrorInputLabelStyle: ElementErrorViewStyle {
     var isHidden: Bool = true
+    var isWarningImageOnLeft: Bool = true
     var backgroundColor: UIColor = .white
     var tintColor: UIColor = .tallPoppyRed
     var text: String = ""
     var font: UIFont = UIFont(graphikStyle: .medium, size: Constants.Style.BillingForm.InputErrorLabel.fontSize.rawValue)
     var textColor: UIColor =  .tallPoppyRed
-    var isWarningImageOnLeft: Bool = true
+    var image: UIImage =  "warning".vectorPDFImage(forClass: CheckoutTheme.self) ?? UIImage()
```

Comment:
Another image imported down a long winded path instead of `UIImage(named: ...`.

[Quick search example](https://stackoverflow.com/a/62685218)